template.yaml

```diff
@@ -115,6 +115,10 @@ Parameters:
     Type: String
     Description: "Specify the ID of an existing user pool client to use that one instead of creating a new one. If specified, then UserPoolArn must also be specified. Note: new callback URL's will be added to the pre-existing user pool client"
     Default: ""
+  UserPoolRequiredGroup:
+    Type: String
+    Description: "Specify a group that a user has to be part of to view private site. If left blank any user in the pool can access the private site. You will need to manually create this group."
```

Comment:
Did you consider adding the group in the CFN template? Would be nice I think.

Reply:
Add a default value to make the param optional

Reply:
I did give it default value, see the line just below this comment:
https://github.com/aws-samples/cloudfront-authorization-at-edge/pull/86/files#diff-363d481fe07d1094db2338998d381b71R121

Yes adding the group in the template sounds good, I'm adding that now.